### PR TITLE
Makes InlineTextStyler usable by itself

### PR DESCRIPTION
**Pull request by jonaskordt**
-addresses #22
-switches order of adding attributes in `MarkdownHeading>>asText`
-`MarkdownInlineTextStyler` now removes any attributes from the text it is given

I think it is important that the `MarkdownInlineTextStyler` is usable by itself, even though we don't use it like that, because otherwise it makes no sense for it to be a TextStyler.

### CODE REVIEW

File: packages/MarkdownEditor-Core.package/MarkdownInlineTextStyler.class/instance/privateStyle..st [hunk 1]
```diff
@@ -2,5 +2,6 @@ parsing
 privateStyle: aText
 	
 	| attributes |
+	aText runs: (RunArray runs: {aText size} values: {{}}).
```
Comment (jonaskordt) [inline packages/MarkdownEditor-Core.package/MarkdownInlineTextStyler.class/instance/privateStyle..st]: This line is the one which removes the previous attributes. I couldn't find a cleaner way to do this. If you have any suggestions please do tell.

Sadly we don't have a `Text` where we can get `runs` from to simply overwrite the `runs` of `aText` like we can in `MarkdownBlockTextStyler>>privateStyle`.

Comment (felix-gohla) [reply packages/MarkdownEditor-Core.package/MarkdownInlineTextStyler.class/instance/privateStyle..st]: How about:
`aText addAttribute: TextEmphasis normal.`?

**Comment by felix-gohla** [comment]
I don't really get the point of this PR... Why should I use it by itself? I read #22. The wording for your PR is confusing. Maybe we could rename it to "Allow InlineTextParser to remove attributes".

**Comment by jonaskordt** [comment]
While you might not have an application for using it by itself right now someone in the future might. There are often occurrences where only inline text should be parsed as allowing Headings and stuff would be too attention grabbing. Since we are creating a `MarkdownInlineTextStyler` we want it to be possible to be used that way. The title of the PR makes perfect sense.

File: packages/MarkdownEditor-Core.package/MarkdownInlineTextStyler.class/instance/privateStyle..st (hunk 1 above)
Comment (jonaskordt) [reply packages/MarkdownEditor-Core.package/MarkdownInlineTextStyler.class/instance/privateStyle..st]: Thanks

File: packages/MarkdownEditor-Tests.package/MarkdownInlineTextStylerTest.class/instance/testPreviousAttributesGetRemoved.st [hunk 2]
```diff
@@ -1,10 +1,10 @@
 tests - parsing
-testPlainStylingKeepsAttributes
+testPreviousAttributesGetRemoved
```
Comment (pixunil) [inline packages/MarkdownEditor-Tests.package/MarkdownInlineTextStylerTest.class/instance/testPreviousAttributesGetRemoved.st:2]: The `BlockTextStyler` should also remove attributes. Could you add a test for it?

**Review by pixunil — APPROVED** [review]

**Review by kiragrammel — APPROVED** [review]

**Pull request merged**